learn/getting_started/quick_start.mdx

```diff
@@ -143,103 +172,14 @@ In the above code sample, the parameter `q` represents the search query. The doc
-
-<CodeSamples id="getting_started_front_end_integration_md" />
-
-### Let's try it!
-
-1. Create an empty file and name it `index.html`
-2. Open it in a text editor like Notepad, Sublime Text, or Visual Studio Code
-3. Copy-paste one of the code samples above—either vanilla JavaScript, Vue 2, or React— and save the file
-4. Open `index.html` in your browser by double-clicking it in your folder
-
-You should now have a working front-end search interface 🚀🔥
-
-## Securing Meilisearch
-
-The Meilisearch API is unprotected by default, making all routes publicly accessible. You can set a master key of at least 16 bytes to protect your instance from unauthorized use:
-
-<Tabs.Container labels={['CLI', 'Environment variable']}>
-
-<Tabs.Content label="CLI">
-
-```bash
-./meilisearch --master-key="MASTER_KEY"
-```
-
-</Tabs.Content>
-
-<Tabs.Content label="Environment variable">
-
-UNIX:
-
-```bash
-export MEILI_MASTER_KEY="MASTER_KEY"
-./meilisearch
-```
-
-Windows:
-
-```bash
-set MEILI_MASTER_KEY="MASTER_KEY"
-./meilisearch
-```
-
-</Tabs.Content>
-</Tabs.Container>
-
-When you launch your Meilisearch instance with a master key, two things happen:
-
-- Your Meilisearch instance is now protected. Aside from the [get health endpoint](/reference/api/health), all subsequent API requests must include a valid API key for [authorization](/reference/api/overview#authorization)
-- Two [default API keys](/learn/security/master_api_keys#using-default-api-keys-for-authorization) are automatically generated
-
-Here's how to use the master key you set to [get all keys](/reference/api/keys#get-all-keys):
-
-<CodeSamples id="authorization_header_1" />
+By default, Meilisearch only returns the first 20 results for a search query. You can change this using the [`limit` parameter](/reference/api/search#limit).
 
-The master key should only be used for retrieving and managing API keys. For regular API calls, such as search, use an API key:
-
-<CodeSamples id="getting_started_communicating_with_a_protected_instance" />
-
-<Capsule intent="warning">
-Accessing the `/keys` route without setting a master key will return an [error](/reference/errors/error_codes#missing-master-key).
-</Capsule>
+## What's next?
 
-To learn more about key management, refer to our [dedicated guide](/learn/security/master_api_keys).
+You now know how to install Meilisearch, create an index, add documents, check the status of an asynchronous task, and make a search request.
 
-## What's next?
+If you'd like to search through the documents you just added using a clean browser interface rather than the terminal, you can do so with [our built-in search preview](/learn/getting_started/search_preview). You can also [learn how to quickly build a front-end interface](/learn/getting_started/front_end_integration) of your own.
```

Comment:
```suggestion
If you'd like to search through the documents you just added using a clean browser interface rather than the terminal, you can do so with [our built-in search preview](/learn/getting_started/search_preview). You can also [learn how to quickly build a front-end interface](/learn/front_end/front_end_integration) of your own.
```